ledger/sandbox/src/main/scala/com/digitalasset/platform/sandbox/SandboxServer.scala

```diff
@@ -63,15 +64,14 @@ object SandboxServer {
   // repeated validation of the sames packages after each reset
   private val engine = Engine()
 
-  def owner(config: SandboxConfig): ResourceOwner[SandboxState] = new ResourceOwner[SandboxState] {
-    override def acquire()(implicit executionContext: ExecutionContext): Resource[SandboxState] = {
-      for {
-        server <- ResourceOwner.forTry(() => Try(new SandboxServer(config))).acquire()
-        state <- server.sandboxState
-        _ <- state.apiServer
-      } yield state
+  def owner(config: SandboxConfig): ResourceOwner[SandboxServer] =
+    new ResourceOwner[SandboxServer] {
+      override def acquire()(implicit executionContext: ExecutionContext): Resource[SandboxServer] =
+        for {
+          server <- ResourceOwner.forTryCloseable(() => Try(new SandboxServer(config))).acquire()
```

Comment:
Maybe, but I think the explicit `acquire` is worth it for things that really should be released. I think this is a bigger question; perhaps all those `ResourceOwner.forXyz` methods should also be available on `Resource`?

Reply:
Does the explicit `acquire` actually change the semantics?

Reply:
Nah. I have a refactoring on the way but it's a big change because I'm making the constructor private, which breaks some stuff, so I'd rather do this in a separate piece of work.